src/Raku/ast/code.rakumod

```diff
@@ -160,7 +160,7 @@ class RakuAST::Code
         # fixed up as needed.
         $block.code_object($code-obj);
 
-        my @compstuff := nqp::getattr($code-obj, Code, '@!compstuff');
+        my @compstuff := nqp::getattr($code-obj, Code, '@!compstuff') || [];
```

Comment:
Is this needed???

Reply:
Yes otherwise it fails to bind at 179: `bindpos_o on VMNull`